markdown/events/2022/bytesize_nf-core-airrflow.md

```diff
@@ -24,3 +24,87 @@ It is our hope that these talks / videos will build an archive of training mater
+Afterwards, there's a step for clonal analysis, which identifies which of the sequences of the BCR belong to which B-cell clone. It assigns the individual BCR sequences and TCR sequences to their specific clone and in the case of the B-cells it can also perform lineage reconstruction of the whole B-cell clone. Finally there's a step for reporting, doing repertoire analysis and reporting, including QC reports by MultiQC. That's the general steps of the pipeline, but now if we look a bit more detailed there is a ton more processes - individual processes that are part of the pipeline. I'm going to explain them a bit in more detail now.
+
+[9:01](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=541)
+Starting for the QCN sequence assembly, the pipeline supports different sequencing protocols, including multiplex PCR, in which the users have to provide the B and C-primer sequences that were used for the amplification, or in the case of 5' RACE, providing the C-primer and the linker sequences for amplification. Both protocols are supported with and without UMI barcodes. The barcodes can also be provided in different configurations. Starting from the raw sequencing data, a sample sheet needs to be provided that contains sample information and the individual FASTQ files for all samples. Depending on if the sequencing protocol includes UMI barcodes or not, there will be some processes but they all start with quality control of the reads with FASTQ, filtering the sequences by quality threshold, masking the primer sequences and if a UMI-based protocol is used, then a consensus is built from all the sequences that have the same UMI barcode. This way, it also allows to correct for the errors as I mentioned before.
+There is an extra procedure, that is employed whenever it's estimated that the length of the UMI barcodes will not be sufficient to cover all of the diversity of the sample and that is bypassed by first clustering all the sequences by similarity and annotating the cluster ID and then two different sequences with the same UMI barcode can also be distinguished this way.
+
+[10:51](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=651)
+After building the consensus, all the sequences that contain the same UMI barcode are collapsed and the count, the number of sequences with the same UMI barcode, is annotated. Other metadata is also annotated as well as the count of duplicate sequences with different UMI barcodes that were collapsed. This can be useful for filtering. After this step there comes the VBJ assignment and filtering step and here it's also possible
```

Comment:
```suggestion
After building the consensus, all the sequences that contain the same UMI barcode are collapsed and the count, the number of sequences with the same UMI barcode, is annotated. Other metadata is also annotated as well as the count of duplicate sequences with different UMI barcodes that were collapsed. This can be useful for filtering. After this step there comes the VDJ assignment and filtering step and here it's also possible
```